src/query/query_manager.c

```diff
@@ -284,9 +282,9 @@ qmgr_allocate_query_entry (THREAD_ENTRY * thread_p, QMGR_TRAN_ENTRY * tran_entry
       tran_entry_p->free_query_entry_list_p = query_p->next;
       pthread_mutex_unlock (&tran_entry_p->mutex);
     }
-  else if (QMGR_MAX_QUERY_ENTRY_PER_TRAN < tran_entry_p->num_query_entries)
+  else if (qmgr_max_query_entry_per_tran < tran_entry_p->num_query_entries)
     {
-      assert (QMGR_MAX_QUERY_ENTRY_PER_TRAN >= tran_entry_p->num_query_entries);
+      assert (qmgr_max_query_entry_per_tran >= tran_entry_p->num_query_entries);
```

Comment:
It's a meaningless check.

```suggestion     
```

Reply:
static_assert works at compile time and QMGR_MAX_QUERY_PER_TRAN parameter is already limited to SHRT (32767).

Reply:
The assert is equivalent to 'assert(false)' because of the if condition in line 285. It seems inappropriate to assert false in this line because this can be reachable when the number of queries exceeds the parameter value.

Reply:
Sorry, 
I misunderstood the assert code.
Assert is a required code.

Reply:
The purpose of static_assert is compiling error.  Any assertion is not neccesary in this location.